daemon/image_windows.go

```diff
@@ -0,0 +1,35 @@
+package daemon // import "github.com/docker/docker/daemon"
+
+import (
+	"github.com/docker/docker/image"
+	"github.com/docker/docker/layer"
+	"github.com/docker/docker/pkg/system"
+	"github.com/pkg/errors"
+)
+
+// GetLayerFolders returns the layer folders from an image RootFS
+func (daemon *Daemon) GetLayerFolders(img *image.Image, rwLayer layer.RWLayer) ([]string, error) {
```

Comment:
Does this have to be exported?

Reply:
Good catch, it doesn't seem like it needs to be.

Reply:
It will need to be in #36240 . So I can change it here and export it later, but that's probably not a good use of time.

Reply:
Ah, thanks for explaining 👍

Reply:
Thanks for the review!